.github/workflows/rustdoc.yml

```diff
@@ -0,0 +1,28 @@
+# This Action triggers a build of Vector's Rustdoc each time a PR is merged
+# to `master`.
+#
+# Netlify account for Rustdoc site: https://app.netlify.com/sites/vector-rustdoc
+# Repo that builds the site: https://github.com/timberio/vector-rustdoc
+# Root URL for Rustdoc site: https://vector-rustdoc.netlify.app
+# Pretty URL for the published docs: https://rustdoc.vector.dev
+name: Publish Vector Rustdoc
+
+# Action is triggered only if changes are merged, not just pushed
+# See also the "if" statement in the job definition
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  trigger-site-build:
```

Comment:
I'm curious if there are any pros/cons for this vs. letting Netlify receive the webhooks directly. Neither seems obviously better or worse. Maybe @sghall has opinions.

Reply:
In this case, we need to trigger a build in a different repository. In the `publish-docs` job (which we're currently not using) we do this in a much more complex way that IMO isn't really necessary here.

Reply:
Aha, gotcha, yeah, I see. I didn't realize this was triggering a build in another repo. And agreed, that seems like a better approach for `publish-docs` as well whenever we bring that back.